dataland-e2etests/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM eclipse-temurin:17.0.2_8-jre-alpine
+
+COPY gradle /app/gradle/
+COPY dataland-e2etests/src /app/dataland-e2etests/src/
+COPY dataland-e2etests/build.gradle.kts /app/dataland-e2etests
+COPY dataland-backend/src /app/dataland-backend/src/
+COPY dataland-backend/build.gradle.kts /app/dataland-backend
+COPY gradlew settings.gradle.kts build.gradle.kts /app/
+COPY testing-shell-scripts/wait_for_backend.sh /app/wait_for_backend.sh
+RUN apk update && apk add bash && apk add curl

```

Comment:
bash brauchst du nicht wenn das wait-for-backend script `#!/bin/sh` bentutzt
Statt curl kannst du wget in dem script benutzen. Dann müssen wir im Dockerfile nichts installieren.

Reply:
Aktuell brauchen wir bash doch im Container, denn im wait_for_backend_and_run_e2etests.sh wird ein bash Befehl ausgeführt:

timeout 240 bash -c "while ! wget http://backend:8080/actuator/health; do echo 'backend server not yet there - retrying in 1s'; sleep 1; done; echo 'backend server responded'" || exit

Ich habe es auch mal getestet ohne bash, dann kommt im e2e-Test-Container an genau dieser Stelle die Fehlermeldung:

timeout: can't execute 'bash': No such file or directory

Reply:
dann würde ich doch mal 
`timeout 240 sh -c "while ! wget http://backend:8080/actuator/health; do echo 'backend server not yet there - retrying in 1s'; sleep 1; done; echo 'backend server responded'" || exit` als Lösung vorschlagen!?

Reply:
ok